test/SlowTests.hs

```diff
@@ -16,24 +16,28 @@ module SlowTests ( main ) where
 import System.LogLevel
 import Test.Tasty
 
+import Data.CAS.RocksDB
+
 -- internal modules
 
 import Chainweb.Graph
+import qualified Chainweb.Test.CutDB
 import qualified Chainweb.Test.MultiNode
 import Chainweb.Version
 
 import qualified Network.X509.SelfSigned.Test
 
 main :: IO ()
-main = defaultMain suite
+main = withTempRocksDb "chainweb-tests" (defaultMain . suite)
 
 loglevel :: LogLevel
 loglevel = Warn
 
-suite :: TestTree
-suite = testGroup "ChainwebSlowTests"
+suite :: RocksDb -> TestTree
+suite rdb = testGroup "ChainwebSlowTests"
     [ Chainweb.Test.MultiNode.test loglevel (TimedConsensus petersonChainGraph twentyChainGraph) 10 120
     , testGroup "Network.X05.SelfSigned.Test"
         [ Network.X509.SelfSigned.Test.tests
         ]
+    , Chainweb.Test.CutDB.tests rdb
```

Comment:
These tests already got added to `testsChainwebTests`. No need to add them here again.

(The tests in ` Network.X509.SelfSigned.Test.tests` are here not because they are slow, but because of nix historically had trouble with executing certain networking tests)